app/models/event.py

```diff
@@ -29,20 +30,25 @@ def __str__(self):
 
     @property
     def noProgram(self):
-        return not self.programEvents.exists()
+        #Ask Anderson 
+        return not self.program_id.exists()
 
-    @property
-    def singleProgram(self):
-        from app.models.programEvent import ProgramEvent
+#anderson ask to comment out
+    # @property
+    # def program(self):
 
-        if self._spCache == "Empty":
-            countPE = list(self.programEvents.select(ProgramEvent, Program).join(Program).execute())
-            if len(countPE) == 1:
-                self._spCache = countPE[0].program
-            else:
-                self._spCache = None
+    #     if self._spCache == "Empty":
+    #         # countPE = list(Event.select( Event, Program).join(Program).execute())
+    #         countPE = list(Event.select(Event, Program).join(Program).execute())
+    #         print("#################")
+    #         print(countPE)
+    #         print("###################")
+    #         if len(countPE) == 1:
+    #             self._spCache = countPE[0].program
+    #         else:
+    #             self._spCache = None
 
-        return self._spCache
+    #     return self._spCache
```

Comment:
Get rid of the function, don't just comment it out. If we need it back we can get it from git.